src/translate/compute.rs

```diff
@@ -26,16 +26,17 @@ pub fn translate_computation(meta: &TranslateMetadata, operation: ArithOp, left:
     match meta.arith_module {
         ArithType::BcSed => {
             let (left, right) = (left.unwrap_or_default(), right.unwrap_or_default());
-            let mut math_lib_flag = true;
-            // Removes trailing zeros from the expression
-            let sed_regex = "/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//";
+            let mut scale = "";
             let op = match operation {
                 ArithOp::Add => "+",
                 ArithOp::Sub => "-",
                 ArithOp::Mul => "*",
-                ArithOp::Div => "/",
+                ArithOp::Div => {
+                    scale = "scale=0;";
```

Comment:
I don't understand the reasoning behind this. Why do we set scale to zero? This will treat `Num` as integer for all division operations:

![image](https://github.com/user-attachments/assets/d84c7594-22cd-4eb3-83bb-ed17cafe8a5d)

Reply:
I saw that in this way the test works in the majority of cases

Reply:
Yeah but we shouldn't sacrifice the precision too. `1/2` should still equal `0.5` and not `0`

Reply:
We should have tests for those cases so we can see if there is an issue or not.